### PR TITLE
BSP-289: fixed unit test that was failing because of timezone

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BSP-289

### Change description ###
Just fix unit test that was failing because of timezones.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.joda.time.DateTimeZone;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.io.File;
-import java.sql.Date;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java]: BTW, we were comparing java.sql.Date and java.util.Date... :o

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java [hunk 2]
```diff
@@ -25,7 +30,7 @@ public void setUp() throws Exception {
     public void shouldCreateOrderRefusalFromJson() {
         assertThat(order.getOrderRefusalAfterText(), is("afterText"));
         assertThat(order.getOrderRefusal(), hasItems("Other"));
-        assertThat(order.getOrderRefusalDate(), is(Date.valueOf("2003-02-01")));
+        assertThat(order.getOrderRefusalDate(), is(parseDate("2003-02-01")));
```
Comment (jvictorcf) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java]: I think I'd put a .toString in front of the date and compare the strings.

Comment (yarpo) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java]: It was returning:

```
Expected : is <Sat Feb 01 00:00:00 CET 2003>
Actual   :<Sat Feb 01 01:00:00 CET 2003>
```

Comment (jvictorcf) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java]: hum. weird...
even with the java.util.Date?
Documentation says it should return in "yyyy-MM-dd" format.
https://docs.oracle.com/javase/8/docs/api/java/sql/Date.html#toString--

Comment (jvictorcf) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java]: Ignore me. I was looking at the java.sql one.

Comment (jvictorcf) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java]: Maybe ".getInstant().toString()" will work. If you think it's worth a try.

Comment (yarpo) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java]: ```order.getOrderRefusalDate().toInstant().toString()```

returns:
`2003-02-01T00:00:00Z`

so timezone issue is still there :/

**Pull request closed without merging**